cp-quickstart/connectors/submit_datagen_users_config_cloud.sh

```diff
@@ -17,10 +17,13 @@ DATA=$( cat << EOF
     "kafka.topic": "users",
     "quickstart": "users",
     "key.converter": "org.apache.kafka.connect.storage.StringConverter",
-    "value.converter": "io.confluent.connect.avro.AvroConverter",
+    "value.converter": "io.confluent.connect.protobuf.ProtobufConverter",
     "value.converter.basic.auth.credentials.source": "$BASIC_AUTH_CREDENTIALS_SOURCE",
     "value.converter.schema.registry.basic.auth.user.info": "$SCHEMA_REGISTRY_BASIC_AUTH_USER_INFO",
     "value.converter.schema.registry.url": "$SCHEMA_REGISTRY_URL",
+    "transforms": "SetSchemaMetadata",
```

Comment:
@rspurgeon will you also be adding an example of this to https://github.com/confluentinc/kafka-connect-datagen/tree/0.3.x/config to highlight the SMT required for ProtobufConverter?

Reply:
There is DEVX-1575 for this task, I'll add your note to that